lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,63 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+import { entitlementApi } from './endpoints';
+
+const HEADERS = {
+  Accept: 'application/json',
+  'Content-Type': 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+}
+
+const requestEntitlements = ({username}) => {
+  return fetch(
+    `${entitlementApi}/?user=${username}`, {
+      credentials: 'same-origin',
+      method: 'get'
+    },
+  );
+}
+
+const createEntitlement = ({username, courseUuid, mode, reason, comments}) => {
```

Comment:
Same here

Reply:
they should both be `action` and should be included in the `support_details` object